src/java.base/share/classes/jdk/internal/classfile/constantpool/InvokeDynamicEntry.java

```diff
@@ -37,6 +38,13 @@ public sealed interface InvokeDynamicEntry
         extends DynamicConstantPoolEntry
         permits AbstractPoolEntry.InvokeDynamicEntryImpl {
 
+    /**
+     * {@return a symbolic descriptor for the call site's invocation type}
+     */
+    default MethodTypeDesc typeSymbol() {
+        return Util.methodTypeSymbol(nameAndType());
+    }
```

Comment:
Similar comment as with CondyEntry -- reduce usage of Util::methodTypeSymbol in this class

Reply:
(Would like to do the same in various ::count methods, but unfortunately MemberRefEntry is too general; would have to rejigger the various XxxRefEntry types.)

Reply:
Which `count` methods are you referring to, Brian?